app/services/language_selector.rb

```diff
@@ -0,0 +1,42 @@
+class LanguageSelector
+  pattr_initialize :literals
+  
+  def labels
+    if found_labels.empty?
+      literals
+    else
+      found_labels
+    end
+  end
+
+  private
+
+  def found_labels
+    @found_labels ||= literals.select do |literal|
+      preferred_labels.include?(literal.language.to_s)
+    end
+  end
+
+  def preferred_labels
+    ["en", "en-us"]
```

Comment:
I'm assuming this is in order of preference - if so, should we prioritize en-us?  Seems like if we're defaulting to the first label, we won't get the specific label as often as the general one.  And maybe that doesn't matter... I really don't know.